torchani/data/new.py

```diff
@@ -0,0 +1,449 @@
+import numpy as np
+import torch
+import functools
+from ._pyanitools import anidataloader
+import importlib
+pkbar_installed = importlib.util.find_spec('pkbar')
```

Comment:
The name `pkbar_installed` sounds like a boolean value. Also, constants should be all upper case please use:
```python
PKBAR_INSTALLED = (importlib.util.find_spec('pkbar') is not None)
```